src/pages/EditMonitor.vue

```diff
@@ -660,10 +660,48 @@
                                 </div>
                             </div>
 
-                            <!-- Ping packet size -->
+                            <!-- Max Packets / Count -->
+                            <div v-if="monitor.type === 'ping'" class="my-3">
+                                <label for="ping-count" class="form-label">{{ $t("Max Packets") }}</label>
+                                <input id="ping-count" v-model="monitor.ping_count" type="number" class="form-control" required :min="ping_count_min" :max="ping_count_max" step="1">
+                                <div class="form-text">
+                                    {{ $t("Number of packets to send before stopping") }}
+                                </div>
+                            </div>
+
+                            <!-- Numeric Output -->
+                            <div v-if="monitor.type === 'ping'" class="my-3 form-check">
+                                <input id="ping_numeric" v-model="monitor.ping_numeric" type="checkbox" class="form-check-input" :checked="monitor.ping_numeric">
+                                <label class="form-check-label" for="ping_numeric">
+                                    {{ $t("Numeric Output") }}
+                                </label>
+                                <div class="form-text">
+                                    {{ $t("If checked, IP addresses will be output instead of symbolic hostnames") }}
```

Comment:
please make sure that all translations are added to `en.json`.
Otherwise they are not avaliable to our translators